docs/features/software-catalog/external-integrations.md

```diff
@@ -66,9 +66,9 @@ have to supply a (unique) name, and accept a connection from the environment
 through which you can issue writes. The rest is up to the individual provider
 implementation.
 
-It is up to you where you put the code for this new processor class. For quick
+It is up to you where you put the code for this new provider class. For quick
 experimentation you could place it in your backend package, but we recommend
-putting all extensions like this in a backend plugin package of their own in the
+putting all extensions like this in a backend module package of their own in the
```

Comment:
Would it make sense to include the command to do this? It would just be `yarn new --select backend-module`

Reply:
Thanks, @awanlin! I've included the command, and since we're in the context of external integrations for the Catalog plugin, I've also included the plugin ID in the command so developers don't have to type it in:
```sh
yarn new --select backend-module --option id=catalog
```
The prompt would look like so:
```
$ yarn new --select backend-module --option id=catalog
? Enter the ID of the module [required] aida

Creating backend module @internal/plugin-catalog-backend-module-aida

 Checking Prerequisites:
  availability  plugins/catalog-backend-module-aida ✔ 
  .
  .
  .
```

Reply:
That's awesome, many thanks!